addons/stock_account/wizard/stock_valuation_layer_revaluation.py

```diff
@@ -16,8 +16,6 @@ def default_get(self, default_fields):
         res = super().default_get(default_fields)
         if res.get('product_id'):
             product = self.env['product.product'].browse(res['product_id'])
-            if product.categ_id.property_cost_method == 'standard':
-                raise UserError(_("You cannot revalue a product with a standard cost method."))
```

Comment:
I don't think it was asked to enable that feature for `standard` `cost_method`, but only be able to display the valuation layers.

Reply:
Hello @clesgow ,

<img width="1896" alt="2024-05-16_16-33-57" src="https://github.com/odoo/odoo/assets/157002399/b61c6b7c-6925-4e9c-8101-813cf71b0d16">

Based on the image provided, we need to add a valuation icon as well as a '+' button to all the products, as I understand. Please let me know if the '+' button should be required for all product or not.

Reply:
You misunderstood, that should still be disabled for `standard` methods.